src/main/java/de/metas/ui/web/window/model/DocumentCollection.java

```diff
@@ -620,7 +617,7 @@ else if (childEntityDescriptors.size() > 1)
 					.retrieveParentDocumentId(rootEntityDescriptor);
 
 			//
-			return DocumentPath.includedDocumentPath(zoomIntoWindowIdEffective, rootDocumentId, childEntityDescriptor.getDetailId(), rowId);
+			return DocumentPath.includedDocumentPath(zoomIntoWindowIdEffective, rootDocumentId, Objects.requireNonNull(childEntityDescriptor.getDetailId()), rowId);
```

Comment:
please use Check.assumeNotNull, so there can be a message with at lest some infos.
or, use `@NonNull` in `includedDocumentPath(...)`

Reply:
using Check.